### PR TITLE
[UI] js/hbs linting: part 1

### PR DESCRIPTION
**Pull request by jmar910**
### Description
+ Lints JS and hbs project files and addresses issues
+ ~20 or so `interactive-element` hbs lint errors to be addressed in subsequent PR

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: ui/app/components/pipeline-editor.hbs [hunk 1]
```diff
@@ -59,7 +60,7 @@
           <div id="editor-container" class="w-full h-full">
             <PipelineEditor::SvgLayer />
             <div id="editor-bg" class="p-12 w-full h-full absolute">
-              <div class="flex items-center space-x-16" style="width: 900px;">
+              <div class="flex items-center space-x-16 w-editor">
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor.hbs:63]: Nice! :sparkles:

File: ui/app/components/pipeline-editor/connector-slide-panel.js [hunk 2]
```diff
@@ -8,19 +8,6 @@ import { run } from '@ember/runloop';
 
 import { validatePresence } from 'ember-changeset-validations/validators';
 
-const TRANSFORM_SUFFIXES = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
-
-function* transformSuffixGenerator() {
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor/connector-slide-panel.js:13]: This function is not needed anymore, correct?

File: ui/package.json [hunk 3]
```diff
@@ -70,7 +73,7 @@
     "ember-test-selectors": "^6.0.0",
     "ember-tooltips": "^3.5.1",
     "ember-truth-helpers": "^3.0.0",
-    "eslint": "^8.7.0",
+    "eslint": "^7.32.0",
```
Comment (jayjayjpg) [inline ui/package.json:76]: I'm wondering if the eslint downgrade is needed or if we could also stick to the `8+` version - what do you think?

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left two more small questions, but this is already good to go! :sparkles:

File: ui/package.json (hunk 3 above)
Comment (jmar910) [reply ui/package.json:76]: It was needed unfortunately. There is a mismatch with eslint and a babel dependency that ember relies on.

Comment (jayjayjpg) [reply ui/package.json:76]: Got it and thank you for outlining!

**Pull request merged**